styleguide.config.js

```diff
@@ -178,6 +178,7 @@ module.exports = { // eslint-disable-line
                                 injectType: 'singletonStyleTag',
                             },
                         },
+                        'css-loader',
```

Comment:
i figured that his was missing by comparing the configuration inside of the `webpack.config.js` with the configuration used here. we use the 'css-loader' before the `MiniCssExtractPlugin` in the `webpack.config.js` here:
https://github.com/sulu/sulu/blob/a8b771793decf61248288dead37407f0950f0690/webpack.config.js#L121-L139